src/client/src/app/services/api.service.ts

```diff
@@ -0,0 +1,15 @@
+import { HttpClient } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class ApiService {
+
+  constructor(private http: HttpClient) 
+  { }
+
+  get<T>(resourceName: string) {
+    return this.http.get<T>('https://localhost:3501/'+ resourceName);
```

Comment:
I used a variable called root for 'http://localhost:3501' so that i can use it again later (maybe for a post function if we need it).... also i think its just 'http://' .. ill ask how that goes

Reply:
Yeah I also have doubt regarding this. I'm not sure if the '/' is needed at the end or not.